src/runtime_src/core/include/ert.h

```diff
@@ -909,6 +975,16 @@ ert_valid_opcode(struct ert_packet *pkt)
     /* header count must match number of commands in payload */
     valid = (pkt->count == (ccdata->command_count * sizeof(uint64_t) + sizeof(struct ert_cmd_chain_data)) / sizeof(uint32_t));
     break;
+  case ERT_START_IPU:
+    skcmd = to_start_krnl_pkg(pkt);
+    /* 1 mandatory cumask + extra_cu_masks + ert_ipu_data */
+    valid = (skcmd->count >= 1+ skcmd->extra_cu_masks + sizeof(struct ert_ipu_data) / sizeof(uint32_t));
+    break;
+   case ERT_START_IPU_PREEMPT:
+    skcmd = to_start_krnl_pkg(pkt);
```

Comment:
This code path is not tested yet, I guess

Reply:
We don't have an end-to-end preemption flow yet.